packages/core/src/common/audioAlerts.ts

```diff
@@ -0,0 +1,191 @@
+import { Howl } from 'howler';
+import { minimumSeverity, testAudioSrc } from './constants';
+
+export class AudioAlertPlayer {
+  private isMute = false;
+  private isPlay = false;
+  private maxVolume = 1.0;
+  private currentlyPlayingSeverity: number = minimumSeverity;
+  private soundID: number | undefined = undefined;
```

Comment:
I don't think you need the initialization part here, since an uninitialized variable will be given the value `undefined` anyway